helpme/templates/helpme/ticket_list.html

```diff
@@ -10,60 +10,20 @@ <h1>
 </div>
 <div class="row no-gutters">
   <a href="{% url 'helpme:faq' %}" class="mr-3">
-    <h5>
+    <h5 class="support-link">
```

Comment:
We should not use custom CSS and make this work with Bootstrap.

Reply:
@renderbox Ok. I updated it to take out the custom css classes, but is it ok to add inline styling occasionally when bootstrap doesn't quite cover what you want to do?